contracts/JobRegistry.sol

```diff
@@ -216,6 +249,33 @@ contract JobRegistry {
         );
     }
 
+    /**
+     * @notice Allows the employer to assign an employee to the job
+     * @param _jobId Job identifier
+     * @param _employeeId Handle for the user
+     */
+
+    function assignEmployeeToJob(uint256 _jobId, uint256 _employeeId) public {
+
```

Comment:
il faut que tu configures ton prettier sur ton vsCode, pour qu'il formate et corrige ces lignes à supprimer

Reply:
Yes i realize it yesterday, it's done